nemo/core/classes/exportable.py

```diff
@@ -172,11 +174,14 @@ def _export(
                         do_constant_folding=do_constant_folding,
                         dynamic_axes=dynamic_axes,
                         opset_version=onnx_opset_version,
+                        export_modules_as_functions=export_modules_as_functions,
                     )
 
                     if check_trace:
                         if isinstance(check_trace, bool):
-                            check_trace_input = [input_example]
+                            check_trace_input = [self.input_module.input_example(max_dim=333),
```

Comment:
Dont hardcode for streaming case, revert

Reply:
These are changes done by @borisfom. Going to ask him check them out. We can try random numbers.

Reply:
reverted back.